src/utils/adjustDateTimes.js

```diff
@@ -13,5 +18,18 @@ export function adjustDateTimes(openDate, closeDate) {
     adjustedCloseDate: adjustedCloseDate.isSame(moment.utc(closeDate))
       ? adjustedCloseDate.subtract(1, 'minute')
       : moment.utc(closeDate),
+    timeSuffixes,
   };
 }
+
+function getTimeSuffix(date) {
+  if (
+    moment.utc(date).format('HH:mm') === '23:59' ||
+    moment.utc(date).format('HH:mm') === '00:00' ||
+    moment.utc(date).format('HH:mm') === '00:01'
+  ) {
+    return ' (Midnight)';
+  } else if (moment.utc(date).format('HH:mm') === '12:00') {
+    return ' (Midday)';
+  } else return '';
+}
```

Comment:
No need to recalculate the formatted date each time, plus a switch might be more favourable over a if else chain here
```suggestion
function getTimeSuffix(date) {
  switch (moment.utc(date).format('HH:mm')) {
    case '23:59':
    case '00:00':
    case '00:01':
      return ' (Midnight)';
    case '12:00':
      return ' (Midday)';
    default:
      return '';
  }
}
```

Reply:
This looks so much nicer 😮